src/marcgrep/destinations/plaintext.clj

```diff
@@ -2,13 +2,19 @@
   (:use clojure.java.io)
   (:require [marcgrep.protocols.marc-destination :as marc-destination])
   (:refer-clojure :exclude [next flush])
-  (:import [org.marc4j.marc Record VariableField DataField ControlField Subfield]
+  (:import [org.marc4j.marc Record VariableField DataField ControlField Subfield Leader]
            [java.io BufferedWriter FileOutputStream]))
 
 
 (def ms-between-flushes 10000)
 
 (defn write-pretty-record [^Record record ^BufferedWriter out included-fields]
+  (when (or (not included-fields)
+            (included-fields "000"))
+    (let [^Leader lead (.getLeader record)]
+      (do (.write out "000 " )
+          (.write out (.marshal lead))
+          (.write out "\r\n"))))
```

Comment:
I think these `(do ...)` forms inside the `let` forms are unnecessary? Maybe I've forgotten something as I'm pretty rusty with Clojure these days but I'm pretty sure `let` allows multiple statements in its body. Not sure why the existing code has it, maybe it was left in accidentally after some refactoring.

```suggestion
      (.write out "000 " )
      (.write out (.marshal lead))
      (.write out "\r\n")))
```